next-app/components/PostPageNav.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'next/link';
+import { useTranslation } from 'next-i18next';
+
+const PostPageNav = ({ nextLink, prevLink }) => {
+  const { t } = useTranslation('post');
+  return (
+    <div className="row PageNavigation mt-4 prevnextlinks d-flex justify-content-between">
+      <div className="col-md-6 rightborder pl-0">
+        {prevLink.header && (
+          <Link href={prevLink.name}>
```

Comment:
```suggestion
          const prevHref = {
              pathname: '/[name]',
              query: {
                  name: prevLink.name,
              },
          };
          <Link href={prevHref}>
```

Так next.js будет строить ссылки в соответствии с текущей локалью - https://nextjs.org/docs/api-reference/next/link#with-url-object

Reply:
не знал про этот момент, я сначала подумал что нужно как то формировать ссылки с учетом языка - но оно работало и так :)